lib/util/fs_operations.js

```diff
@@ -0,0 +1,38 @@
+const { promises: Fs } = require('fs')
+const _isString = require('lodash/isString')
+
+const pathExists = async (path) => {
+  try {
+    await Fs.access(path)
+    return true
+  } catch {
+    return false
+  }
+}
+
+const createDir = async (path) => {
+  try {
+    await Fs.mkdir(path)
```

Comment:
should be recursive?

Reply:
`await Fs.mkdir(path, { recursive: true })`

Reply:
done